config.js

```diff
@@ -34,6 +34,14 @@ module.exports = {
   // See `bufferSize`.
   flushDelaySeconds: 30,
 
+  // If paths are present in this array, then these paths will be ignored before
+  // `samplingRate` based decisions are made. Paths must include a leading
+  // forward slash and be of the form:
```

Comment:
Looking at `FilterPolicy#matches` implementation, it doesn't seem like we actually depend on the leading slash as a requirement. Am I reading the code right?